Library/Homebrew/test/test_diagnostic.rb

```diff
@@ -204,20 +204,6 @@ def test_check_for_symlinked_cellar
     HOMEBREW_CELLAR.mkpath
   end
 
-  def test_check_for_autoconf
-    MacOS::Xcode.stubs(:installed?).returns true
```

Comment:
Sorry, please ignore me. My suggestion doesn't make any sense. Since the `doctor` check is being removed, there's nothing that could be tested. When first reading this, I somehow thought this was only testing `MacOS::Xcode.provides_autotools?`.